src/test/java/easytests/api/v1/controllers/QuestionsControllerTest.java

```diff
@@ -1,13 +1,193 @@
+                                .with(topic, new JsonSupport().with(id, questionsModels.get(0).getTopic().getId()))
+                                .with(answers, new JsonSupport()
+                                    .with(new JsonSupport()
+                                        .with(id, questionsModels.get(0).getAnswers().get(0).getId())
+                                        .with(txt, questionsModels.get(0).getAnswers().get(0).getTxt())
+                                        .with(right, questionsModels.get(0).getAnswers().get(0).getRight())
+                                        .with(number, questionsModels.get(0).getAnswers().get(0).getSerialNumber()))
+                                    .with(new JsonSupport()
+                                        .with(id, questionsModels.get(0).getAnswers().get(1).getId())
+                                        .with(txt, questionsModels.get(0).getAnswers().get(1).getTxt())
+                                        .with(right, questionsModels.get(0).getAnswers().get(1).getRight())
+                                        .with(number, questionsModels.get(0).getAnswers().get(1).getSerialNumber()))
+                                )
+                        )
+                        .with(new JsonSupport()
+                                .with(id, questionsModels.get(1).getId())
+                                .with(text, questionsModels.get(1).getText())
+                                .with(type, questionsModels.get(1).getQuestionType().getId())
+                                .with(topic, new JsonSupport().with(id, questionsModels.get(1).getTopic().getId()))
+                                .with(answers, new JsonSupport()
+                                    .with(new JsonSupport()
+                                        .with(id, questionsModels.get(1).getAnswers().get(0).getId())
+                                        .with(txt, questionsModels.get(1).getAnswers().get(0).getTxt())
+                                        .with(right, questionsModels.get(1).getAnswers().get(0).getRight())
+                                        .with(number, questionsModels.get(1).getAnswers().get(0).getSerialNumber()))
+                                    .with(new JsonSupport()
+                                        .with(id, questionsModels.get(1).getAnswers().get(1).getId())
+                                        .with(txt, questionsModels.get(1).getAnswers().get(1).getTxt())
+                                        .with(right, questionsModels.get(1).getAnswers().get(1).getRight())
+                                        .with(number, questionsModels.get(1).getAnswers().get(1).getSerialNumber()))
+                                )
+                        ).build()
+                ))
+                .andReturn();
+    }
+
+    @Test
+    public void testListNotFound() throws Exception {
+        int topicIdParamValue = 5;
+
```

Comment:
where is mock ?

Reply:
What mock? I see no list method with mock here

Reply:
Mock for returning null on fetching question.